src/modules/io/Elsa.IO.Http/Elsa.IO.Http.csproj

```diff
@@ -7,18 +7,21 @@
         <PackageTags>elsa module io http</PackageTags>
     </PropertyGroup>
 
-    <ItemGroup>
-        <ProjectReference Include="..\Elsa.IO.Compression\Elsa.IO.Compression.csproj" />
-        <ProjectReference Include="..\Elsa.IO\Elsa.IO.csproj" />
-    </ItemGroup>
-
     <ItemGroup>
         <PackageReference Include="Microsoft.Extensions.Http" />
         <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
     </ItemGroup>
 
     <ItemGroup Label="Elsa" Condition="'$(UseProjectReferences)' != 'true'">
         <PackageReference Include="Elsa.Workflows.Management" />
+        <PackageReference Include="Elsa.IO" />
+        <PackageReference Include="Elsa.IO.Compression" />
+    </ItemGroup>
+
+    <ItemGroup Label="Elsa" Condition="'$(UseProjectReferences)' == 'true'">
+        <ProjectReference Include="..\..\..\..\..\elsa-core\src\modules\Elsa.Workflows.Management\Elsa.Workflows.Management.csproj" />
+        <ProjectReference Include="..\Elsa.IO\Elsa.IO.csproj" />
+        <ProjectReference Include="..\Elsa.IO.Compression\Elsa.IO.Compression.csproj" />
```

Comment:
Only the elsa-core project references should be conditionally included. The extension projects themselves are already part of the current repo, and so should always be referenced.